src/enums/index.ts

```diff
@@ -17,4 +17,5 @@ export * from './roleTypes'
 export * from './routeNames'
 
 // external enums
-export { CorpTypeCd, CorpClass, SummaryActions } from '@bcrs-shared-components/enums'
+export { SummaryActions } from '@bcrs-shared-components/enums'
+export { CorpTypeCd, CorpClass } from '@bcrs-shared-components/corp-type-module'
```

Comment:
Why did you change this?

Reply:
When we updated to `"@bcrs-shared-components/enums": "1.0.14"` it wasn't recognizing the `CorpTypeCd` & `CorpClass`, it needed to be explicit. 
I'm not sure what changed, i hadn't been working with the enums, i just updated this at your request 'while i was in there'

Reply:
Thanks for the update.

That's weird, since the shared index.ts definitely exports those. Must be another one of those cases where the system doesn't recognize those to be the same (for which we have a ticket).